### DISCUSSION TITLE
ON to OFF delay one Minute and turn AC Relay back on

### DISCUSSION THREAD

**Discussion by mischa-t** [arendst/Tasmota#15574] [Q&A] [answered]
Hello all,
i have a Gosund 112 and flashed it with Tasmota.
All works fine, with a single push on the button you torn on or off your ac relay.
With a dubble action on the button you turn oon or off your 5v usb.

I would like to change the AC Relay, i want to use it as an Power Interrupt for my router.
The AC Relay after Buttonstate change shoud go from On to OFF delay one Minute an turn AC Relay back on.

I quite doent know how to start,
can somebody help.

Thanks

**Comment by sfromis** [comment, marked as answer]
You could use `PowerOnState 5` and having set a `PulseTime` for the one-minute off period.

**Comment by mischa-t** [comment]
Super, thanks.
It works good
